external-dependencies.gradle

```diff
@@ -3,7 +3,7 @@ ext {
     jersey_version = '2.23.2'
     jackson_version = '2.8.3'
 
-    verifiableLog = 'verifiable-log:verifiable-log:0.1.67'
+    verifiableLog = 'verifiable-log:verifiable-log:0.2.0000'
```

Comment:
So maybe we can put `verifiableLog = 'verifiable-log:verifiable-log:0.2.+'` over here to take the latest. VL does not change that often really and usually if it does then we do rebuild and bump the number over here. They are closely coupled when it comes to triggering changes so it may be ok to put +